src/datasets/jetstream.md

```diff
@@ -0,0 +1,243 @@
+
+# Statistics tables
+
+The statistics tables contain statistical summaries of their
+corresponding aggregate tables. These tables are suitable for plotting
+directly without additional transformations.
+
+Statistics tables are named like:
+
+`statistics_<slug>_{day, week, overall}_<index>`
+
+A view is also created that concatenates all statistics tables for an
+experiment of a given period type, named like:
+
+`statistics_<slug>_{daily, weekly, overall}`
+
+Statistics tables have the schema:
+
+<table>
+<thead>
+<tr class="header">
+<th>Column name</th>
+<th>Type</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>`segment`</td>
+<td>STRING</td>
+<td>The segment of the population being analyzed. “all” for the entire population.</td>
+</tr>
+<tr class="even">
+<td>`metric`</td>
+<td>STRING</td>
+<td>The slug of the metric, like `active_ticks` or `retained`</td>
+</tr>
+<tr class="odd">
+<td>`statistic`</td>
+<td>STRING</td>
+<td>The slug of the statistic that was used to summarize the metric, like “mean” or “deciles”</td>
+</tr>
+<tr class="even">
+<td>`parameter`</td>
+<td>NUMERIC (decimal)</td>
+<td>A statistic-dependent quantity. For two-dimensional statistics like “decile,” this represents the x axis of the plot. For one-dimensional statistics, this is NULL.</td>
+</tr>
+<tr class="odd">
+<td>`comparison`</td>
+<td>STRING</td>
+<td>If this row represents a comparison between two branches, this row describes what kind of comparison, like `difference` or `relative_uplift`. If this row represents a measurement of a single branch, then this column is NULL.</td>
+</tr>
+<tr class="even">
+<td>`comparison_to_branch`</td>
+<td>STRING</td>
+<td>If this row represents a comparison between two branches, this row describes which branch is being compared to. For simple A/B tests, this will be “control.”</td>
+</tr>
+<tr class="odd">
+<td>`ci_width`</td>
+<td>FLOAT64</td>
+<td>A value between 0 and 1 describing the width of the confidence interval represented by the lower and upper columns. Valued at 0.95 for 95% CIs.</td>
+</tr>
+<tr class="even">
+<td>`point`</td>
+<td>FLOAT64</td>
+<td>The point estimate of the statistic for the metric given the parameter.</td>
+</tr>
+<tr class="odd">
+<td>`lower`</td>
+<td>FLOAT64</td>
+<td>The lower bound of the confidence interval for the estimate.</td>
+</tr>
+<tr class="even">
+<td>`upper`</td>
+<td>FLOAT64</td>
+<td>The upper bound of the confidence interval for the estimate.</td>
+</tr>
+<tr class="odd">
+<td>`window_index`</td>
+<td>INT64</td>
+<td>(views only) A base-1 index reflecting the analysis window from which the row is drawn (i.e. day 1, day 2, …).</td>
+</tr>
+</tbody>
+</table>
+
+Each combination of `(segment, metric, statistic, parameter, comparison,
+comparison_to_branch, ci_width)` uniquely describes a single data
+point.
+
+To extract the mean of `active_hours` for each branch from a weekly
+statistics view v, you could run the query:
```

Comment:
This is pretty abstract. Can you make this more concrete and relatable by using an example from a real table?

Reply:
I don't want to promise that any specific table exists but I added a more realistic looking slug.